### ISSUE TITLE
Multiple-times recurred TODOs

### ISSUE THREAD

**Issue by Ekleog** [Ekleog/todiff#9]
### base.txt
```
2018-04-08 foo due:2018-04-08 rec:1d
```

### todo.txt
```
x 2018-04-08 2018-04-08 foo due:2018-04-08 rec:1d
x 2018-04-08 2018-04-08 foo due:2018-04-09 rec:1d
2018-04-08 foo due:2018-04-10 rec:1d
```

### Result
```
No new tasks.

Changed tasks:
 → 2018-04-08 foo due:2018-04-08 rec:1d
    → Completed on 2018-04-08
    → Recurred (from 2018-04-08) and completed on 2018-04-08
    → Copied and postponed (strict) by 2 days
```

### Expected result
Something like this?
```
No new tasks.

Changed tasks:
 → 2018-04-08 foo due:2018-04-08 rec:1d
    → Completed on 2018-04-08
    → Recurred (on 2018-04-08, was due 2018-04-08) and completed on 2018-04-08
    → Recurred (on 2018,04-08, was due 2018-04-09) and completed on 2018-04-08
```

**Comment by Nadrieril** [comment]
I think the behavior you want is for `rec:+1d`. If so, then https://github.com/Ekleog/todiff/pull/16 fixes this

**Comment by Ekleog** [comment]
Fixed thanks to #16. Thank you!
